addons/website_slides_forum/controllers/main.py

```diff
@@ -6,8 +6,21 @@
 
 
 class WebsiteSlidesForum(WebsiteSlides):
+
+    def _prepare_channel_values(self, **kwargs):
+        forum_type = kwargs.get('forum_type')
+        if forum_type:
+            if forum_type == 'forum':
+                forum = request.env['forum.forum'].create({
+                    'name': kwargs.get('name')
+                })
+                kwargs['forum_id'] = forum.id
+            kwargs['allow_comment'] = forum_type == 'comment'
```

Comment:
Is allow_comment depending on either there is a forum or not ? I thought allow_comment was for rating/review/posting comment ?

Reply:
At frontend slide_channel creation, they can only choose one option : 
- Forum
- Comment
- Nothing
In backend, they can choose both.